client/examples/tutorial.rs

```diff
@@ -185,3 +189,56 @@ fn asset_registration_test(config: &Configuration) -> Result<(), Error> {
     // Finish the test successfully
     Ok(())
 }
+
+fn asset_minting_test() -> Result<(), Error> {
+    // BEGIN FRAGMENT: mint_asset
+    use std::str::FromStr;
+    use iroha_data_model::Value;
+    use iroha_data_model::IdBox;
+    use iroha_data_model::prelude::MintBox;
+    use iroha_data_model::prelude::AssetId;
+    use iroha_data_model::prelude::AssetDefinitionId;
+    use iroha_data_model::prelude::AccountId;
+
+    let roses = AssetDefinitionId::from_str("rose#wonderland")
+        .expect("Valid, because the string contains no whitespace, has a single '#' character and is not empty after");
+    let alice: AccountId = "alice@wonderland".parse()
+        .expect("Valid, because the string contains no whitespace, has a single '@' character and is not empty after");
+
+    let _mint_roses = MintBox::new(
+        Value::U32(42),
+        IdBox::AssetId(AssetId::new(roses.clone(), alice.clone())),
+    );
+    // END FRAGMENT
+
+    // Finish the test successfully
+    Ok(())
+}
+
+fn asset_burning_test() -> Result<(), Error> {
```

Comment:
Is it intentional that you do nothing with `MintBox/BurnBox`?
We need to submit instruction in order to execute it.

Reply:
Yes, I believe we should test the data structures here, not the whole Iroha. It allows to run tests both locally and not, and this will be used in the doctests. I was thinking about modifying the `dst-parser`, so some comments are uncommented and the submit instructions are added, too. I am not sure yet.

Reply:
UPD: now that I've re-read the code, I have noticed the request are sent in the other parts and I will update it accordingly.